ui/app/routes/vault/cluster/clients/counts.ts

```diff
@@ -86,10 +87,7 @@ export default class ClientsCountsRoute extends Route {
   async model(params: ClientsCountsRouteParams) {
     const { config, versionHistory } = this.modelFor('vault.cluster.clients') as ModelFrom<ClientsRoute>;
     // only enterprise versions will have a relevant billing start date, if null users must select initial start time
-    let startTime = null;
-    if (this.version.isEnterprise && this._hasConfig(config)) {
-      startTime = getUnixTime(config.billingStartTimestamp);
-    }
+    const startTime = setStartTimeQuery(this.version.isEnterprise, config);
```

Comment:
Although this logic is fairly simple and only used in two places, I wanted to consolidate it so that setting the start time was handled the same in both places the `/activity` log is queried. See Jira VAULT-25315 for more context around the bug introduced by not checking if the version was enterprise introduced.